lib/friendly_id/finder_methods.rb

```diff
@@ -59,14 +59,47 @@ def potential_primary_key?(id)
     end
 
     def first_by_friendly_id(id)
-      find_by(friendly_id_config.query_field => id.downcase)
+      find_by(friendly_id_config.query_field => parse_friendly_id(id))
+    end
+
+    # Parse the given value to make it suitable for use as a slug according to
+    # your application's rules.
+    #
+    # This method is not intended to be invoked directly; FriendlyId uses it
```

Comment:
I'd say that this paragraph is not needed. I'd leave it out. I don't think there's much value to have this extra paragraph here and maybe it's also incorrect, as maybe someone might want to use `parse_friendly_id` in some other situation...

Reply:
If this paragraph is removed, I'd also remove `However, ` from the next paragraph

Reply:
The default implementation is just a passthrough accessor to `value`, so I would argue that this paragraph is warranted. It also has [precedence](https://github.com/norman/friendly_id/blob/f5733d024da4760fba64ce9ce6d3ad5fc0ee5e25/lib/friendly_id/slugged.rb#L256)

Reply:
Ah interesting. Thanks!